### PR TITLE
[examples] whisper aishell

### PR DESCRIPTION
**Pull request by xingchensong**

### CODE REVIEW

**Comment by xingchensong** [comment]
Training time of 40 epochs

image

**Review by Mddct — DISMISSED** [review]

File: examples/aishell/whisper/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+# Performance Record
+
+## Whisper-largev3 (conv1d2, full-parameter tuning) Result
+
+* Feature info: using log_mel_spectrogram feature, no cmvn, no speed perturb
+* Training info: bf16, deepspeed stage1, activation checkpointing, batch dynamic12000, acc_grad 4, 8 * 3090 gpu, 40 epochs (about 14 hours)
+* Decoding info: ctc_weight 0.3, average_num 5
+* Git hash: TBD
```
Comment (robin1001) [inline examples/aishell/whisper/README.md:30]: shall we add the whipser baseline on the test?

Comment (xingchensong) [reply examples/aishell/whisper/README.md:30]: sure

**Review by robin1001 — APPROVED** [review]

**Comment by srdfjy** [comment]
Hi, can the 16GB V100 be used for training?

**Comment by xingchensong** [comment]
> Hi @xingchensong , can the 16GB V100 be used for training?

8 * V100? V100 dose not support BF16, but I think it's ok to train fp32 model under deepspeed stage-3

**Comment by srdfjy** [comment]
Yes, 8 * V100, each card with 16GB memory. Can training be done with 16GB memory?

**Comment by xingchensong** [comment]
> Yes, 8 * V100, each card with 16GB memory. Can training be done with 16GB memory?

even 8 * 2080ti works, please follow this setting https://github.com/wenet-e2e/wenet/pull/2173#issuecomment-1831240833

**Comment by srdfjy** [comment]
> > Yes, 8 * V100, each card with 16GB memory. Can training be done with 16GB memory?
> 
> even 8 * 2080ti works, please follow this setting [#2173 (comment)](https://github.com/wenet-e2e/wenet/pull/2173#issuecomment-1831240833)

THX!

**Pull request merged**